CHANGES.rst

```diff
@@ -5,7 +5,9 @@
 4.4.2 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Add support for Python 3.8.
+
+- Drop support for Python 3.4.
```

Comment:
The next version should be 4.5.0 because of this.

Reply:
Just last week I listened to a podcast and SemVer was also a topic. And since then I changed my mind. But yep, better be kinda consistent about it.

But I am even unsure about naming the next version 4.5 or 4.5.0, as the change log both shows a 4.4 but a 4.3.0.

Reply:
[PEP 440](https://www.python.org/dev/peps/pep-0440/#final-releases), the version number pep, says that a trailing `.0` is implicitly assumed for a final release of `X.Y`, so `X.Y` == `X.Y.0`. But explicit is better than implicit, so I prefer to always use the full three digits.

Reply:
@jamadden Thanks for claryfing! I even did not know that there is a PEP for it.

Bumped version to 4.5.0